### PR TITLE
Add textarea form component

### PR DESCRIPTION
**Pull request by rohithbalaji123**
Part of #612 

### Description
Add a textarea form field to allow users to input multiple lines.
### Screenshots
image

### CODE REVIEW

File: frontend/src/component/form/TextAreaField.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  onChange?: (value: string) => void;
+  onBlur?: () => void;
+}
+
+export class TextAreaField extends Component<IProps, any> {
+  render() {
+    return (
+      <textarea
+        className={'text-area'}
+        rows={this.props.rows}
```
Comment (byliuyang) [inline frontend/src/component/form/TextAreaField.tsx:17]: I wonder whether we can make this fill in it's parent with 
```
{
    width: 100%;
    height: 100%;
}
```

File: frontend/src/component/form/TextField.scss [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 .text-field {
   @include default-font();
-  border: 2px solid #9e9e9e;
+  border: 2px solid $color-text-field-border;
```
Comment (byliuyang) [inline frontend/src/component/form/TextField.scss:7]: Thank you!

File: frontend/src/component/form/TextAreaField.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  test('should render with correct placeholder', () => {
+    const { container } = render(<TextAreaField placeholder={'placeholder'} />);
+    const textarea = container.querySelector('textarea');
+
+    expect(textarea).toBeTruthy();
+    expect(textarea!.placeholder).toMatch('placeholder');
+  });
+
+  test('should fire onChange when input changes', () => {
```
Comment (rohithbalaji123) [inline frontend/src/component/form/TextAreaField.spec.tsx:26]:
```suggestion
  test('should trigger onChange when input changes', () => {
```

File: frontend/src/component/form/TextAreaField.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    const textarea = container.querySelector('textarea');
+
+    expect(textarea).toBeTruthy();
+    fireEvent.change(textarea!, { target: { value: 'text' } });
+
+    expect(onChange).toHaveBeenCalledTimes(1);
+    expect(onChange).toHaveBeenLastCalledWith('text');
+  });
+
+  test('should fire onBlur when textarea blurs', () => {
```
Comment (rohithbalaji123) [inline frontend/src/component/form/TextAreaField.spec.tsx:38]:
```suggestion
  test('should trigger onBlur when textarea blurs', () => {
```

**Pull request merged**